sdk/tools/azure-openrewrite/src/main/resources/META-INF/rewrite/migrationRecipes/core/com.azure.core.cryptography.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------
+# Migration Recipes for azure.core.cryptography
+# --------------------------------------------------------------------
+type: specs.openrewrite.org/v1beta/recipe
+name: com.azure.openrewrite.recipe.azure.core.cryptography
+displayName: Migrate from azure.core.cryptography to next generation stack
+description: This recipe migrates azure.core.cryptography package to the next generation stack.
+recipeList:
+
+  # --------------------------------------------------------------------
+  # AsyncKeyEncryptionKey
+  # --------------------------------------------------------------------
+
+  # TODO: Async Library. Port to sync.
+
+  # --------------------------------------------------------------------
+  # AsyncKeyEncryptionKeyResolver
+  # --------------------------------------------------------------------
+
+  # TODO: Async Library. Port to sync.
+
+  # --------------------------------------------------------------------
+  # KeyEncryptionKey
+  # --------------------------------------------------------------------
+
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: com.azure.core.cryptography.KeyEncryptionKey
+      newFullyQualifiedTypeName: com.azure.v2.core.cryptography.KeyEncryptionKey
+
+  # TODO: Migrate Methods and Fields
+
+  # --------------------------------------------------------------------
+  # KeyEncryptionKeyResolver
+  # --------------------------------------------------------------------
+
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: com.azure.core.cryptography.KeyEncryptionKeyResolver
+      newFullyQualifiedTypeName: com.azure.v2.core.cryptography.KeyEncryptionKeyResolver
```

Comment:
We don't have cryptography package in v2. We should remove this recipe as we don't expect to have this package in v2.